tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "esModuleInterop": true, // Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'.
     "forceConsistentCasingInFileNames": true, // Disallow inconsistently-cased references to the same file.
     "outDir": "lib",
+    "lib": ["ES2020.Promise", "dom"],
```

Comment:
Allow support for `Promise.allSettled`. Once we added a library we needed to add "dom" also. If we don't add "dom" we get the error " Cannot find name 'URL'. Did you mean 'url'" when running `yarn install`

"dom" adds support for the library URL.

Reply:
thanks for the explainer